package.json

```diff
@@ -15,7 +15,7 @@
     "license:add": "$PWD/scripts/add-license.sh",
     "license:delete": "$PWD/scripts/delete-license.sh",
     "ci:version-check": "yarn version check",
-    "ci:build": "yarn workspaces foreach --exclude=sc --exclude=tina-cloud-starter --exclude=tina-tailwind-sidebar-demo  --exclude=test --exclude=demo -pt run build",
+    "ci:build": "yarn workspaces foreach --exclude=sc --exclude=tina-cloud-starter --exclude=tina-tailwind-sidebar-demo  --exclude=test --exclude=demo -pt --topological-dev run build",
```

Comment:
This _should_ be covered already with `-pt` (`t` being the `topological-dev` shorthand), was that still not working for you?

Reply:
I dont think it is. https://yarnpkg.com/cli/workspaces/foreach#options
`-t` or `--topological`: `Run the command after all workspaces it depends on (regular) have finished`
while
`--topological-dev`: does the same thing for dev dependencies.  I was getting a build error when not using ` --topological-dev`

Reply:
I for this PR to build I also need it to build the dev dependencies in order

Reply:
Ah gotcha, ok works for me